server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteSegmentStatsFromNodesStatsIT.java

```diff
@@ -173,15 +177,15 @@ private void assertZeroRemoteSegmentStats(RemoteSegmentStats remoteSegmentStats)
         assertEquals(0, remoteSegmentStats.getDownloadBytesStarted());
         assertEquals(0, remoteSegmentStats.getDownloadBytesSucceeded());
         assertEquals(0, remoteSegmentStats.getDownloadBytesFailed());
-        assertEquals(0, remoteSegmentStats.getMaxRefreshBytesLag());
+        assertEquals(0, remoteSegmentStats.getTotalRefreshBytesLag());
```

Comment:
here as well, you should be adding another line for total refresh bytes lag?

Reply:
Added maxLagBytes check as well